reconcile/ocm_clusters.py

```diff
@@ -83,153 +255,73 @@ def run(dry_run, gitlab_project_id=None, thread_pool_size=10):
         settings=settings,
         init_provision_shards=True,
     )
+
+    # current_state = ocm_state
```

Comment:
Is this meant to be a comment for readers, or was this code that was commented out?

Reply:
I changed the comment to not seem like a code line.